pyInterface/userInterface/eigenvectorLikelihoodSlices.py

```diff
@@ -128,7 +130,9 @@
 		graphLikeli = ROOT.TGraph()
 		for p in xrange(-50, 51):
 			ratio = (p/50.0) * math.sqrt(eigenVectorsMinuit[par][1])
-			pars = minimum + ratio * eigenVectorsMinuit[par][0]
+			step = ROOT.TVectorD(eigenVectorsMinuit[par][0])
+			step *= ratio
+			pars = minimum + step
```

Comment:
Could you add a TODO marker here reminding use to properly write this once we all use ROOT versions where this is possible to do?